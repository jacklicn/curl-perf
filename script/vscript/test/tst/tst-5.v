
var @arr

arr[0] = 0
arr[1] = 1
arr[2] = 2
arr[3] = 3

var b = (arr[0] + 1) * arr[3]

print(b)

print( arr[0] )

// iterate over array
var i = 0

while i < length(arr)
    print( arr[ i ] )
    i = i + 1
end



