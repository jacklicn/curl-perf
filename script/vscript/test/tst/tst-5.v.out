*** syntax ok ***
*** semantics and code generation ok ***

>3

>0

>0

>1

>2
